ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockTransactionSelector.java

```diff
@@ -188,7 +188,7 @@ private TransactionSelectionResult evaluateTransaction(final Transaction transac
             .getTransactionPriceCalculator()
             .price(transaction, processableBlockHeader.getBaseFee());
     if (minTransactionGasPrice.compareTo(actualMinTransactionGasPriceInBlock) > 0) {
-      LOG.trace(
+      LOG.info(
```

Comment:
I don't think INFO is appropriate here, I would probably expect to see this either as a WARNING or DEBUG log entry

Reply:
I spoke with @macfarla about this since she created the issue. I was thinking about implementing WARNING, but I think she mentioned that INFO was more appropriate for this.

Reply:
I think in essence the issue here (correct me if I'm wrong) is that if you're not in DEBUG or TRACE mode when you run Besu, you want to be able to see some kind of message alerting you to the fact that the transaction didn't go through because the gasPrice was lower than the minimum required.

Reply:
Let's wait for Sally to clarify this one, dismiss my review if we're going ahead with INFO :)